src/components/src/main/java/org/apache/jmeter/visualizers/backend/influxdb/HttpMetricsSender.java

```diff
@@ -48,11 +48,23 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Influxdb sender base on The Line Protocol. The Line Protocol is a text based
- * format for writing points to InfluxDB. Syntax : <measurement>[,<tag_key>=
- * <tag_value>[,<tag_key>=<tag_value>]] <field_key>=<field_value>[,<field_key>=
- * <field_value>] [<timestamp>] Each line, separated by the newline character,
- * represents a single point in InfluxDB. Line Protocol is whitespace sensitive.
+ * InfluxDB sender base on The Line Protocol.
+ * <p>
+ * The Line Protocol is a text based format for writing points to InfluxDB.
+ * Syntax:
+ * <pre>
+ * weather,location=us-midwest temperature=82 1465839830100400200
+ * |    -------------------- --------------  |
```

Comment:
The formatting seems a bit odd. Is it expected?

Reply:
It was a little off, but now I've corrected it to be the same as the original doc: https://docs.influxdata.com/influxdb/v1.7/write_protocols/line_protocol_tutorial/